linera-rpc/src/grpc/conversions.rs

```diff
@@ -614,6 +614,7 @@ impl TryFrom<api::ChainInfoQuery> for ChainInfoQuery {
             request_fallback: chain_info_query.request_fallback,
             request_sent_certificate_hashes_by_heights,
             request_sent_certificate_hashes_in_range: None,
+            create_network_actions: chain_info_query.create_network_actions.unwrap_or(true),
```

Comment:
This is what maintains backwards compatibility, isn't it?

The default value in the _code_ might as well be `false`? That would save a lot of `no_network_actions` calls, and replace them with only one or two `with_network_actions` calls, wouldn't it?